### PR TITLE
[GEOS-6865] Allow wms extension capabilities providers to customize the root crs list and the layer scale ranges

### PR DESCRIPTION
**Pull request by aaime**
This one follows up discussion on the mailing list. It is meant to be merged on the master branch once 2.7.x has been cut off

### CODE REVIEW

File: src/wms/src/main/java/org/geoserver/wms/capabilities/Capabilities_1_3_0_Transformer.java [hunk 1]
```diff
@@ -988,26 +992,30 @@ protected void handleLayer(final LayerInfo layer) throws IOException {
...
+                    scaleDenominators = provider
+                            .overrideScaleDenominators(layer, scaleDenominators);
+                }
+
+                if (scaleDenominators.getMinimum() != 0.0) {
+                    element("MinScaleDenominator", String.valueOf(scaleDenominators.getMinimum()));
+                }
+
+                if (scaleDenominators.getMaximum() != Double.POSITIVE_INFINITY) {
+                    element("MaxScaleDenominator", String.valueOf(scaleDenominators.getMinimum()));
```
Comment (aaime) [inline src/wms/src/main/java/org/geoserver/wms/capabilities/Capabilities_1_3_0_Transformer.java]: Should be getMaximum() here

**Comment by jodygarnett** [comment]
2.7.x has been cut, we should be able to clean up this pull request and merge.

**Comment by aaime** [comment]
Hi Jody, yep, I just need to find the time to make a couple of fixes.

**Comment by jodygarnett** [comment]
Did you want to back port this one as well?

**Comment by aaime** [comment]
No need, at least in the short term

**Pull request merged**